src/main/java/com/github/invictum/panels/FloatingPanel.java

```diff
@@ -0,0 +1,36 @@
+package com.github.invictum.panels;
+
+import net.serenitybdd.core.pages.WebElementFacade;
+import org.openqa.selenium.By;
+
+import java.util.List;
+
```

Comment:
We need to redefine all methods that used panel object. I. e. findBy, findAll, etc.